tests/test_ai_reports.py

```
@@ -0,0 +1,321 @@
+        assert result["tokens_used"] == 350
+        mock_cache_save.assert_called_once()
+
+    @patch("api.services.ai_service.AIReportService._call_anthropic")
+    @patch("api.services.ai_service.AIReportService._save_to_cache")
+    @patch("api.services.ai_service.AIReportService._query_cache")
+    @patch("api.services.ai_service.AIReportService._fetch_game_context")
+    @patch("api.services.ai_service.AIReportService._fetch_game_scoring_plays")
+    @patch("api.services.ai_service.AIReportService._fetch_game_team_stats")
+    def test_generate_game_summary_flow(
+        self, mock_team_stats, mock_scoring, mock_game_ctx,
+        mock_cache_query, mock_cache_save, mock_anthropic,
```

Ruff (0.15.2)

[warning] Unused method argument: `mock_cache_save`

(ARG002)